tiger/src/lib.rs

```diff
@@ -53,7 +55,7 @@ impl TigerState {
         let (a, b, c) = self.0;
         let (mut a, mut b, mut c) = (Wrapping(a), Wrapping(b), Wrapping(c));
 
-        let mut data: [u64; 8] = unsafe { mem::uninitialized() };
+        let mut data: [u64; 8] = [0; 8];
```

Comment:
What was your reasoning when replacing this usage of `unsafe`?

Reply:
Because mem::uninitialized is deprecated since rust 1.39, and as said in [this issue](_https://github.com/RustCrypto/hashes/issues/57_) it not requred for optimal performance.

Reply:
https://github.com/RustCrypto/hashes/issues/57